Translation corrections (#430)
07th-mod/higurashi-console-arcs: Update/ura_008.txt

//r巴の脳裏に、既視感が蘇って映像を呼び起こす。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "巴の脳裏に、既視感が蘇って映像を呼び起こす。",

> 			NULL, "A sense of deja-vu recalled an image in the back of Tomoe's mind.", Line_Normal);

Reviewer: ```suggestion
			NULL, "A sense of deja-vu awakened an image in the back of Tomoe's mind.", Line_Normal);
```

//rその代償に、彼女の手の甲の皮膚が酷くえぐられて血が吹き出し、引き裂かれるような激痛が走った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その代償に、彼女の手の甲の皮膚が酷くえぐられて血が吹き出し、引き裂かれるような激痛が走った。",

> 			NULL, "In exchnage, Natsumi's nails gouged the skin on the back of her hands hard enough to gush blood, and severe pain ran through her like she was being torn apart.", Line_Normal);

Reviewer: ```suggestion
			NULL, "In exchange, Natsumi's nails gouged the skin on the back of her hands hard enough to spout blood, and severe pain ran through her like she was being torn apart.", Line_Normal);
```

//r夢の中の話なのかもしれない。私が今いる世界とは違うところで起きた、無関係の出来事なのかもしれない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "夢の中の話なのかもしれない。私が今いる世界とは違うところで起きた、無関係の出来事なのかもしれない。",

> 			NULL, "Maybe this is all story from a dream. These may be unrelated events that happened in a completely different world.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Maybe this is all a story from a dream. These may be unrelated events that happened in a completely different world.", Line_Normal);
```
